src/main/java/uk/gov/register/RegisterApplication.java

```diff
@@ -99,7 +99,9 @@ public void run(RegisterConfiguration configuration, Environment environment) th
         DBI jdbi = dbiFactory.build(environment, configuration.getDatabase(), "postgres");
 
         Flyway flyway = configuration.getFlywayFactory().build(configuration.getDatabase().build(environment.metrics(), "flyway_db"));
-
+        flyway.setBaselineVersionAsString("0");
+        flyway.migrate();
```

Comment:
no worries, this had to be brought back because #95 wasn't merged at that time, now it will disappear :)